packages/backend-common/src/reading/types.ts

```diff
@@ -18,6 +18,32 @@ import { Logger } from 'winston';
 import { Config } from '@backstage/config';
 import { ReadTreeResponseFactory } from './tree';
 
+/**
```

Comment:
here i mainly moved things around a little - it's been bugging me that `ReadTreeResponse` was at the top of the file instead of `UrlReader` :)

Only relevant change should be the rename of `ReadTreeResponseFile`